src/components/core.ts

```diff
@@ -158,9 +158,9 @@ export default class Core {
     _.setLogLevel(this.config.logLevel);
 
     /**
-     * If initial Block's Tool was not passed, use the Paragraph Tool
+     * If default Block's Tool was not passed, use the Paragraph Tool
      */
-    this.config.initialBlock = this.config.initialBlock || 'paragraph';
+    this.config.defaultBlock = this.config.defaultBlock || 'paragraph';
```

Comment:
we need to save the previous deprecated name for backward capability issues

Reply:
I do as u said, try to keep both the `initialBlock` and the `defaultBlock` and push a new commit ASAP.